interfaces/builtin/steam_support.go

```diff
@@ -205,6 +205,8 @@ deny /usr/bin/{chfn,chsh,gpasswd,mount,newgrp,passwd,su,sudo,umount} x,
 /run/host/usr/sbin/ldconfig* ixr,
 /run/host/usr/bin/localedef ixr,
 /var/cache/ldconfig/** rw,
+/sys/module/nvidia/version r,
```

Comment:
On a side note, should this also be added to `opengl` interface and or `nvidia-drivers-support` ?

Reply:
This is potentially useful for anything that talks to any of the Nvidia proprietary drivers for accelerated stuff (OpenGL, Vulkan, VDPAU, CUDA, anything in that general theme), so I think it would be good to allow it in any AppArmor profile that allows access to the device nodes. Reading this pseudo-file seems to be the easiest way to query which version of the kernel module is in use.

This is important because Nvidia requires that the kernel module and the user-space drivers should be in lockstep. You can potentially have several parallel branches of the user-space drivers available in different paths, but only the one that matches the kernel version will work, and the others are just dead weight (but potentially become useful after a reboot into a different kernel).

For comparison with non-Snap technologies: Flatpak packages each major version of the user-space drivers separately, and uses `/sys/module/nvidia/version` to find out which one it ought to be using to match the kernel.